swift-sdk/Internal/FlexMessagingManager.swift

```diff
@@ -7,5 +7,17 @@ import Foundation
 class FlexMessagingManager: IterableFlexMessagingManagerProtocol {
     init() {
         ITBInfo()
+        
+        
     }
+    
+    deinit {
+        ITBInfo()
+    }
+    
+    func getMessages(placementId: String) -> [IterableFlexMessage] {
```

Comment:
Have to make a note that: 
This will make it more like inapp messages where getMessages dont trigger sync. It will just give back what was fetched until now. 
Actual fetching of flex messages will still be somewhere else in this approach